src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResource.kt

```diff
@@ -87,9 +92,46 @@ class AuditResource(
     @RequestParam who: String? = null,
     @RequestParam what: String? = null,
   ): Page<AuditDto> = auditService.findPage(pageable, who, what)
+
+  @PreAuthorize("hasRole('ROLE_AUDIT')")
```

Comment:
@mikehalmamoj @Mjwillis I'm thinking this might a good candidate for using `read` and `write` scope. 
I think this API to add an event should be write and the getter should be `read` - interestingly I think all clients should have `write` but very few read.

Or alternatively maybe no role? Just need a valid token. In fact I think that ,might be better, the ability to write an audit entry just requires a valid JWT ? What do you all think?

Reply:
I like the idea about read and write scope.  It's quite interesting because it's upside down - normally write would be the less permissive scope but not in this case.

Not sure about dropping the role though. What if supplier X turns on their fancy new service and drop 20GB onto the audit table in the first hour?  Would we want to revoke their access until they get their house in order?